src/progress/pct/v10/pctCompile.p

```diff
@@ -394,6 +400,12 @@ FUNCTION CheckIncludes RETURNS LOGICAL (INPUT f AS CHARACTER, INPUT TS AS DATETI
             ASSIGN TimeStamps.ttFile = IncFile
                    TimeStamps.ttFullPath = SEARCH(IncFile).
             ASSIGN TimeStamps.ttMod = getTimeStampF(TimeStamps.ttFullPath).
+            IF lignoreIncludes AND CAN-DO(cignoreIncludes, IncFile) THEN /* fake older date, to be not relevant */

```

Comment:
tiny remark 
I don't now if getting a timestamp takes a long time but I wouldn't execute it if I'm about to ignore the result. I'd move the getTimeStampF to the else of "IF lignoreIncludes AND CAN-DO".

On the other hand it might be better if you add a flag ignore to the TimeStamps table. It could be easier for debugging purposes to see that a timestamp is ignored than to see a fake timestamp.

just my 0,02€